framework/core/src/Notification/MailableInterface.php

```diff
@@ -14,12 +14,24 @@
 interface MailableInterface
 {
     /**
-     * Get the name of the view to construct a notification email with.
+     * Get the names of the views to construct a notification email with.
+     *
+     * To provide the best experience for the user, Flarum expects both a `text` and `html` view.
+     *
+     * @return array{
+     *     text: string,
+     *     html: string
+     * }
      */
-    public function getEmailView(): string|array;
+    public function getEmailViews(): array;
 
     /**
      * Get the subject line for a notification email.
      */
     public function getEmailSubject(TranslatorInterface $translator): string;
+
+    /**
+     * Get the serialized type of this activity.
+     */
+    public static function getType(): string;
```

Comment:
looking at the code, i think it would be better to replace the `MailableInterface` typing in the send email job and notification mailer classes with `MailableInterface&BlueprintInterface`